### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by Konkss** [LibreHardwareMonitor/LibreHardwareMonitor#1126] [Ideas]
Would love A toggle under "View" or for Libre to detect if system is already set to a dark mode.

**Comment by tofuschnapspraline** [comment]
Yes Please! I would love to run this alongside my other apps without my eyes starting to bleed.

**Comment by roke-julian-lockhart** [comment]
> https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/discussions/1126#discussion-5429966

@KonkaBass, this has been suggested at https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/116#issue-480238721.

**Reply by ebergerly** [reply]
You're referencing a request that was made in 2019, which is now 5 years ago. And people in the discussion couldn't figure how to do a simple change to the Form background colors and Font colors, primarily due to the lack of basic documentation in code that has been around for over 14 years. 

Yeah, it will happen...

**Comment by Konkss** [comment]
https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/pull/847

Looks like someone made a working branch. Would still like to see main branch support, but this will do donkey.
https://github.com/MRog40/LibreHardwareMonitorDark/releases

**Reply by MaverickBlue** [reply]
Excellent, installing it now. I hope the dev keeps it updated. It's the only thing I hate about Libre Hardware Monitor. Well that and the fact that the graphing line is really hard to see because it's soo thin, but apparently that's a changeable option in the fork too! You've made my day!
